index.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html lang="en">
+   <head>
+        <meta charset="UTF-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>title</title>
+        <link rel="stylesheet" href="styles.css">
+    </head>
+<body>
+    <header id="header">
+      <img id="header-img" src="https://d92mrp7hetgfk.cloudfront.net/images/sites/misc/SmallerMicroverselogo_revised/original.jpg?1581442029" alt="Logo Microverse" />
+      <nav id="nav-bar">
+        <ul>
+          <li class="nav-link">Features</li>
+          <li class="nav-link">How it works</li>
+          <li class="nav-link">Pricing</li>
+        </ul>
+      </nav>
+      <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="Roman Carnival Overture Op. 9 for Five Trombones" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+      <form id="form" action="https://www.freecodecamp.com/email-submit" method="POST">
+        <label>Email <input id="email" placeholder="Enter your email" type="email"></input>
```

Comment:
Please make sure that the email field accepts only emails.

Reply:
- [ ] @jlberbesi kindly try always to include an explanation, a "why", for each requested change. For example:

> - **_This is a well-explained request_**: Why don't you use an article tag for the .first-article element instead of a div, that would improve the HTML structure and make the page more accessible
> - _**This request lacks a "why":**_ Why don't you use an article tag here instead of a div?

Reply:
Comment was updated from initial version of it.